packages/core/src/modules/proofs/repository/PresentationExchangeRecord.ts

```diff
@@ -0,0 +1,4 @@
+export enum PresentationRecordType {
+  Indy = 'indy',
+  JsonLD = 'JSON-LD',
+}
```

Comment:
Where are we using these for? I think they can be removed.

Reply:
We are using this enum for formatServiceMap in V2ProofService so I will rename `JsonLD` with `PresentationExchange`